src/main/java/com/epam/lowcost/util/DateFormatter.java

```diff
@@ -8,12 +8,18 @@
 @UtilityClass
 public class DateFormatter {
 
-    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss");
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+    private final DateTimeFormatter customFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd 00:00:00");
```

Comment:
Why you use additional format ? Why you don't use one common format ? I think you can deal with it

Reply:
deal with it